docs/modules/reference/pages/billing/subscription/user-subscription.adoc

```diff
@@ -0,0 +1,119 @@
+= Manage Users
+
+Payara Cloud allows you to have multiple users accessing your subscription.
+In order to do so you invite the user into subscription via email.
+Subsequently, they would register or use their existing Payara Cloud account and get access to the subscription.
+
+== Roles
+
+We recognize multiple roles of users in Payara Cloud:
+
+Billing Manager::
+Users in this role have access to all billing and payment information related to the subscription.
+They can also invite new users to subscription and alter their roles.
+They do not necessarily have permissions to deploy applications deployed to Payara Cloud.
+
+Administrator::
+Administrators have full permissions to work deploy and configure applications deployed to Payara Cloud.
+They can also invite new user to subscription and alter their roles.
+
+Billing Manager with Administrative Privilege::
+This role is union of two roles above, granting you all the permissions.
+When you sign up to Payara Cloud, this is the role you have in your subscription.
+
+Member::
+Member has restricted access to subscription's applications. Namely, they cannot create, delete or modify namespaces.
+
+=== Role Requirements
+At any time a subscription must have at least one user in Billing Manager role and in Administrator role.
+This condition can also be satisfied by having single user in role Billing Manager with Administrative Privilege.
+
+== Manage Subscription Users Screen
+
+User Management activities are performed after navigating to btn:[Manage Users]
+
+.Manage Users Screen
+image::billing/subscription/user-subscription/manage-users.svg[Screenshot of Manage Users screen,100%,opts=interactive]
+
+[checklist]
+* ❶ <<Inviting New User,Invite New User>>
+* ❷ <<Roles>> of respective user
+* ❸ Status of the user
+* ❹ Additional actions are available in drop down menu
+
+=== Inviting New User
+
+First, click btn:[Add User].
+
+image::billing/subscription/user-subscription/add.png[Screenshot of Add User page]
+
+Fill in their email address and the role they should have in the subscription once they accept it.
+An option to grant Administrative privileges to Billing Manager is only enabled when Billing Manager option is checked.
+Only a Billing Manager can grant or remove Billing Manager roles in the subscription.
+
+By clicking btn:[Add User] an email will be sent to provided email address.
+User's email will be present in list with status _Pending_.
+
+image::billing/subscription/user-subscription/email.png[Screenshot of invitation email,65%]
+
+Following link btn:[Signup to Payara Cloud] will lead to signup prompt.
+
+image::billing/subscription/user-subscription/signup.svg[Screeenshot of signup screen,100%,opts=interactive]
+
+The email used for registering does *not* need to match the email invitation was sent to.
```

Comment:
The invitation link is basically one-time secret, first authenticated actor to use it is the one that gets the access.

We need to state this because if you sent an invitation email to me `@payara.fish` and I would log in with my github account, you'd get my email as `github@dudits.net`. And it multiple times was coming around as bug, when it is a natural property of the process :)